src/policy/rbf.h

```diff
@@ -1,34 +0,0 @@
-// Copyright (c) 2016-2020 The Bitcoin Core developers
-// Distributed under the MIT software license, see the accompanying
-// file COPYING or http://www.opensource.org/licenses/mit-license.php.
-
-#ifndef BITCOIN_POLICY_RBF_H
-#define BITCOIN_POLICY_RBF_H
-
-#include <txmempool.h>
-
-/** The rbf state of unconfirmed transactions */
-enum class RBFTransactionState {
-    /** Unconfirmed tx that does not signal rbf and is not in the mempool */
-    UNKNOWN,
```

Comment:
Nit: it would be nice if e4c04c0409cd734d5b02997cb190e6c383d39f11 were split into 2 commits (1) eliminate "unknown" and (2) refactor to remove `RBFTransactionState`.